plugins/services/src/js/components/SDKPlanTable.tsx

```diff
@@ -0,0 +1,217 @@
+import { Trans } from "@lingui/macro";
+
+import * as React from "react";
+import { Table, Column, TextCell, HeaderCell } from "@dcos/ui-kit";
+import { WidthArgs } from "@dcos/ui-kit/dist/packages/table/components/Column";
+
+import {
+  flattenServicePlanPhases,
+  ServicePlan,
+  ServicePlanElement,
+  NO_DATA_SERVICE_PLAN_ELEMENT
+} from "#PLUGINS/services/src/js/types/ServicePlan";
+import {
+  formatServicePlanStatus,
+  ServicePlanStatus
+} from "#PLUGINS/services/src/js/types/ServicePlanStatus";
+
+import Icon from "#SRC/js/components/Icon";
+import { Tooltip } from "reactjs-components";
+
+const getStatusTooltip = (data: ServicePlanElement): React.ReactNode | null => {
+  const { status } = data;
+
+  switch (status) {
+    case "ERROR":
+      return <Trans render="span">Execution experienced an error.</Trans>;
+    case "WAITING":
+      return <Trans render="span">Execution has been interrupted.</Trans>;
+    case "PENDING":
+      return (
```

Comment:
Is there a reason why this has "()", while error and waiting don't?

Reply:
The reason for this is that the length of the html text is too long for one line so it gets moved to the next line which makes the `()` necessary.